starlite/cache/__init__.py

```diff
@@ -1,5 +1,8 @@
 from .base import Cache, CacheBackendProtocol
-from .redis_cache_backend import RedisCacheBackend, RedisCacheBackendConfig
 from .simple_cache_backend import SimpleCacheBackend
 
-__all__ = ("CacheBackendProtocol", "SimpleCacheBackend", "RedisCacheBackend", "RedisCacheBackendConfig", "Cache")
+__all__ = (
+    "CacheBackendProtocol",
+    "SimpleCacheBackend",
+    "Cache",
+)
```

Comment:
The reason I removed redis backend is that both redis and memcached are optional dependencies and if they're not installed - importing anything from `starlite.cache.*` will fail